rotkehlchen/db/filtering.py

```diff
@@ -461,10 +461,32 @@ def make(
         )
         filter_query = cast('TradesFilterQuery', filter_query)
         filters: List[DBFilter] = []
-        if base_asset is not None:
-            filters.append(DBAssetFilter(and_op=True, asset=base_asset, asset_key='base_asset'))
-        if quote_asset is not None:
-            filters.append(DBAssetFilter(and_op=True, asset=quote_asset, asset_key='quote_asset'))
+        if base_assets is not None:
+            if len(base_assets) == 1:
+                filters.append(
+                    DBAssetFilter(and_op=True, asset=base_assets[0], asset_key='base_asset'),
+                )
+            else:
+                filters.append(
+                    DBMultiStringFilter(
+                        and_op=True,
+                        column='base_asset',
+                        values=[asset.identifier for asset in base_assets],
+                    ),
+                )
+        if quote_assets is not None:
+            if len(quote_assets) == 1:
+                filters.append(
+                    DBAssetFilter(and_op=True, asset=quote_assets[0], asset_key='quote_asset'),
+                )
+            else:
+                filters.append(
+                    DBMultiStringFilter(
+                        and_op=True,
```

Comment:
are you sure it should be AND? Wouldn't that mean quote asset should be ETH AND ETH2?

Hmm looking at the implementation it does not matter since `MultisringFilter` uses the `IN` operator so nothing is either ANDed or ORed but this should be either not provided, or set to false in all DBMultistringFilter entries in this PR since it's really confusing when reading the code.